instructure/instructure-ui
# chore: fix docs hot reload not working + convert webpack config to esm

INSTUI-4324

test plan:

- pull this pr, delete you node modules and run npm install + bootstrap
- run `npm run dev` and check if you update any source file, the dev server should reload
- check if you can find any file that doesn't trigger a reload when changed
